website/src/components/articles/ArticleSection.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+import TopNews from "./TopNews.vue";
+import Article from "./MiniArticle.vue";
```

Comment:
Import name should match file name to improve readability of code
```suggestion
import MiniArticle from "./MiniArticle.vue";
```